packages/components/src/RichTextEditor/RichTextEditor/_docs/RichTextEditor.stories.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from "react"
+import { Meta, StoryObj } from "@storybook/react"
+import { EditorContentArray } from "~components/RichTextEditor/types"
+import { RichTextEditor, RichTextEditorProps } from "../index"
+
+const meta = {
+  title: "Components/RichTextEditor",
+  component: RichTextEditor,
+  parameters: {
+    chromatic: { disable: false },
+  },
+  args: {
+    labelText: "Rich Text",
+    defaultValue: [],
+    onChange: () => undefined,
+  },
+} satisfies Meta<typeof RichTextEditor>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Playground: Story = {
```

Comment:
Potential improvement later is to add a drop down for `controls` to let our consumer add items to the toolbar in the playground story